cvat/apps/dataset_manager/formats/yolo.py

```diff
@@ -20,12 +23,21 @@
 from cvat.apps.dataset_manager.util import make_zip_archive
 
 from .registry import dm_env, exporter, importer
+from .transformations import SetKeyframeForEveryTrackShape
 
 
-def _export_common(dst_file, temp_dir, instance_data, format_name, *, save_images=False):
+def _export_common(
+    dst_file: str,
+    temp_dir: str,
+    instance_data: ProjectData | CommonData,
+    format_name: str,
+    *,
+    save_images: bool = False,
+    **kwargs
+):
     with GetCVATDataExtractor(instance_data, include_images=save_images) as extractor:
         dataset = Dataset.from_extractors(extractor, env=dm_env)
-        dataset.export(temp_dir, format_name, save_images=save_images)
+        dataset.export(temp_dir, format_name, save_images=save_images, **(kwargs or {}))
```

Comment:
```suggestion
        dataset.export(temp_dir, format_name, save_images=save_images, **kwargs)
```

`def(**kwargs)` always produces a dict.

```python
>>> def x(**kw): print(kw)
>>> x()
{}
>>> x(y=5)
{'y': 5}
```